server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -100,12 +100,14 @@ public Invocations getInvocations() {
             .deprecated() // TODO: remove in 16
             .handleWith(this::getHealth)
             .invocation().methods(GET, HEAD).path("/v2/container/health")
+            .deprecated()
             .handleWith(this::getHealth)
 
             .invocation().methods(GET, HEAD).anonymous(true).path("/v2/cache-managers/{name}/health/status")
             .deprecated() // TODO: remove in 16
             .handleWith(this::getHealthStatus)
             .invocation().methods(GET, HEAD).anonymous(true).path("/v2/container/health/status")
+            .deprecated()
```

Comment:
Maybe we shouldn't deprecate this method after all... https://github.com/infinispan/infinispan-operator/pull/2277/files#r2096384224

Reply:
@jabolina  I amended your commit to remove this line. We can reconsider in the future if needed.

Reply:
Thanks, @ryanemerson!